apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,29 +625,31 @@ var ConversationView = {
     // to slide correctly. Bug 1009541
     this.cancelEdit();
 
-    if (Navigation.isCurrentPanel('thread')) {
```

Comment:
in that commit, you could check if there is a Threads.active (but right, this is going in the opposite way to what Steve is doing in https://bugzilla.mozilla.org/show_bug.cgi?id=1167144 :D). Initially I wanted to try to remove all isCurrentPanel calls but I couldn't. I think when the views will be split it will be much easier though.

so all in all, no opinion against resurrecting it; I could just as well not removing it in this patch